### ISSUE TITLE
Identify and tag rules recommended for AAP

### ISSUE THREAD

**Issue by ganeshrn** [ansible/ansible-lint#1829]
##### Summary

Identify and tag rules to be recommended for Ansible automation platform (AAP) example FQCN for plugin names
to provide an opinionated way to develop ansible automation (--tag aap)

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version

ansible-lint --version

```

- ansible installation method: one of source, pip, OS package
- ansible-lint installation method: one of source, pip, OS package

##### STEPS TO REPRODUCE

```console (paste below)

```

##### Desired Behaviour

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behaviour

Please give some details of what is actually happening.
Include a [minimum complete verifiable example] with:
- playbook
- output of running ansible-lint
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`

```paste below

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by ssbarnea** [comment]
Closing as done, as that is the production profile.
